KsApi/MockService.swift

```diff
@@ -750,12 +750,6 @@
       return producer(for: self.fetchErroredUserBackingsResult)
     }
 
-    internal func fetchGraph<A>(
-      query _: NonEmptySet<Query>
-    ) -> SignalProducer<A, GraphError> where A: Decodable {
```

Comment:
The last reference to `GraphError` in this file is the `fetchCategory` method on line 1235. I don't see it being called anywhere so I think we can delete that too.

Reply:
Deleted `fetchCategory`, good call was not being used. But `GraphError` needs to still be in `GraphSchema` for the reason above.